contracts/interfaces/IERC20Permit.sol

```diff
@@ -6,6 +6,31 @@ import { IERC20 } from "./IERC20.sol";
 /// @title Interface of the ERC20 standard with the addition of permit functionality outlined in EIP 2612
 interface IERC20Permit is IERC20 {
 
+    /**
+     * @dev    Returns the signature domain separator.
+     * @return domainSeparator_ The domain for the contract.
+     */
+    function DOMAIN_SEPARATOR() external view returns (bytes32 domainSeparator_);
+
+    /**
+     * @dev    Returns the block.chainid when the contract was instantiated.
+     * @return chainId_ The block.chainid when the contract was instantiated.
+     */
+    function INITIAL_CHAIN_ID() external view returns (uint256 chainId_);
```

Comment:
Can we remove `ALL_CAPS` from some or all of these functions (unless we need to adhere to any EIPs)

Reply:
Yeah, the all caps should be used only for constant values, right?